examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p><em>Suggested change:</em></p><p> </p><p>Config for ResNet34 with 10 classes. Named expression&nbsp;<code>C</code>&nbsp;allows substituting variable value at model initialization time.&nbsp;Read more about Batchflow model configuration in <a href="https://github.com/analysiscenter/batchflow/blob/master/examples/tutorials/05_creating_CNN.ipynb" target="_blank">05_creating_CNN tutorial</a>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>